src/l500/l500-options.cpp

```diff
@@ -339,4 +339,57 @@ namespace librealsense
                "Values are rounded to whole meter values and are between 3 meters and 9 meters. "
                "Max range refers to the center 10% of the frame.";
     }
+
+    void ir_reflectivity_option::set(float value)
+    {
+        if (value == 1.0f)
+        {
+            auto &max_usable_range_option = _l500_depth_dev->get_depth_sensor().get_option(RS2_OPTION_ENABLE_MAX_USABLE_RANGE);
```

Comment:
I think we should do this only AFTER we've checked that all the other options are acceptable...

Reply:
👍